src/interactive.js

```diff
@@ -0,0 +1,39 @@
+import Methods from './utils.js';
+
+export default class Interactive {
+static toggleCompleted = (id, curStatus) => {
+  const todoList = Methods.getLocalStorageData();
+  todoList[id].completed = curStatus;
+  Methods.setLocalStorageData(todoList);
+  Methods.showTodoItems();
+}
+
+static addCheckEvent = () => {
+  document.querySelectorAll('.checkbox').forEach((box) => box.addEventListener('change', () => {
+    let id;
```

Comment:
Try to declare variables at the top of your file so they are easy for someone reading your code to see.